index.html

```diff
@@ -0,0 +1,69 @@
+<!doctype html>
+<html>
+    <head>
+        <title>PHP Cornwall</title>
+        <meta name="viewport" content="width=device-width, initial-scale=1">
+        <meta name="description" content="PHP Cornwall Community Group">
+        <meta name="keywords" content="PHP, Wordpress, Cornwall, Web Dev">
+        <meta name="theme-color" content="#dee4ff">
+        <link rel="stylesheet" type="text/css" href="assets/css/output.css">
+        <link rel="stylesheet" type="text/css" href="assets/css/normalize.css">
+        <link href="https://fonts.googleapis.com/css?family=Inconsolata:400,700|Montserrat:400,600&display=swap" rel="stylesheet">
+        <link rel="shortcut icon" href="/assets/images/computer.png" />
+    </head>
+    <body>
+        <header id="header">
+            <a href="/"><h1>PHP Cornwall</h1></a>
+        </header>
+        <main>
+            <section class="text-image-container">
+                <div class="ti-text">
+                    <p>PHP Cornwall is a new user group, aimed at bringing together the extensive PHP community in Cornwall.</p>
+                    <h2>The First meeting</h2>
+                    <p>PHP Cornwall is so new, there hasn't even been a meeting yet!
+                        <br><br>
+                        We'd like to talk to you, the developers of Cornwall and find out exactly what you want from a user group.
+                        <br><br>
+                        For now, the easiest place to get hold of us is on <a href="https://twitter.com/cornwallphp" target="_blank">Twitter</a></p>
```

Comment:
When using `target="_blank"` we need to think of the security implications. Take a look at [this link](https://mathiasbynens.github.io/rel-noopener/) for some more info on this.

Reply:
Ah right, so I've added `rel="noopener"` to these links, that'll make them more secure right?

Reply:
Exactly, make sure you read that page to understand the reasoning behind it though. It's a very good example of how it can be used!

Reply:
Nice, okay, I'll have a read!